packages/dapp/.prettierignore

```diff
@@ -1,3 +1,7 @@
 dist/
 .next/
 fixtures/
+node_modules/
+../contracts/test/dollar/BondingV2.t.sol
+../contracts/test/dollar/UbiquityAlgorithmicDollar.t.sol
+../contracts/test/helpers/LiveTestHelper.sol
```

Comment:
Not sure about this.

Reply:
Why ontract test files should be included in dapp/.prettierignore???